charts/linkerd2-cni/templates/cni-plugin.yaml

```diff
@@ -112,6 +112,12 @@ rules:
 - apiGroups: [""]
   resources: ["pods", "nodes", "namespaces", "services"]
   verbs: ["list", "get", "watch"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["delete"]
+- apiGroups: ["events.k8s.io"]
+  resources: ["events"]
+  verbs: ["create"]
```

Comment:
Can this RBAC be conditioned on the feature being enabled? It's a pretty invasive change.

Reply:
Good catch, I missed that.